support-docs/README.md

```diff
@@ -244,10 +244,342 @@ The table below gives a description of the properties used to specify a switchin
 |package|Package definition of the switching regulator|Object| |
 |complianceList|List of compliances met by the switching regulator component|Array| |
 
+### 3.10	 Hardware
 
+TODO: not sure what separates this category from others? "hardware" is a bit generic sounding.

```

Comment:
Please advice, I am not sure how to document this section.

Reply:
Since Hardware is more of a category and not a specific component, I'm ok not calling it here, since you're already adding the components description.

Reply:
I took out this TODO and just left it blank then, let me know if you'd like me to format it in a different way.